### DISCUSSION TITLE
Disable alert based on time interval

### DISCUSSION THREAD

**Discussion by stonemonkey** [datalust/seq-tickets#1723] [Help] [answered]
Hi,

I would like to set an alert, but somehow disable it (not to run) between certain hours e.g. in the night 01.00 to 5.00. Is that possible?

The use case would be that if an event is not seen for 15mins then fire the alarm, but skip the night time. I have it set for the first part. However, I don't know how or if it is possible to disable it for the night.

![alert](https://user-images.githubusercontent.com/2823090/208113182-c8874379-ca84-4301-b25c-4cd7c0709890.png)

Regards,
/Costin

**Comment by nblumhardt** [comment, marked as answer]
Hi Costin; it's unfortunately a bit on the hacky side, but adding to the `having` clause can do it:

```sql
count = 0 and DatePart(Now(), 'hour', 1h) < 18 and DatePart(Now(), 'hour', 1h) >= 7
```

(Where `1h` is your time zone offset from GMT, which you could also compute with `OffsetIn('Europe/Stockholm', now())`)

Let me know if this helps!

**Reply by stonemonkey** [reply]
Thanks.
That will do for the moment.
/Costin
